Flora/Flora/View Controllers/CommentCell.m

```diff
@@ -0,0 +1,45 @@
+//
+//  CommentCell.m
+//  Flora
+//
+//  Created by Kleo Ku on 8/3/22.
+//
+
+#import "CommentCell.h"
+#import "DateTools/DateTools.h"
+
+@implementation CommentCell
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+    
+    self.profileImage.layer.masksToBounds = false;
+    self.profileImage.layer.cornerRadius = self.profileImage.frame.size.width/2;
+    self.profileImage.clipsToBounds = true;
+    self.profileImage.layer.borderWidth = 0.05;
+}
+
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
+    [super setSelected:selected animated:animated];
+
+    // Configure the view for the selected state
+}
+
+- (void)setComment:(Comment *)comment {
+    _comment = comment;
+    [comment fetchIfNeededInBackgroundWithBlock:^(PFObject * _Nullable object, NSError * _Nullable error) {
+        self.text.text = comment.text;
```

Comment:
do we need a 'reuse check' here?  Since we aren't sure that the cell is on this comment anymore...